diagnostics-app/src/app/rma/test-viewer/audit-page/audit-page.component.ts

```diff
@@ -25,32 +25,27 @@ export class AuditPageComponent extends BaseTestComponent {
   testStatistics = new TestStatistics();
 
   UpdateTestStatistics() {
-    this.ngZone.run(() => {
-      this.testStatistics = new TestStatistics();
-      for (const [index, testConfig] of this.testList.entries()) {
-        if (!testConfig.enabled) {
-          this.testStatistics.disabledTests.push(index);
-          continue;
-        }
-        this.testStatistics.enabledTests.push(index);
-        const testInfo = this.testResults[index];
-        if (!testInfo) {
-          this.testStatistics.unranTests.push(index);
-        }
-        if (testInfo && !testInfo.passed) {
-          this.testStatistics.failedTests.push(index);
-        }
-        if (testInfo && testInfo.passed) {
-          this.testStatistics.passedTests.push(index);
-        }
+    this.testStatistics = new TestStatistics();
+    for (const [index, testConfig] of this.testList.entries()) {
+      if (!testConfig.enabled) {
+        this.testStatistics.disabledTests.push(index);
+        continue;
       }
-    });
+      this.testStatistics.enabledTests.push(index);
+      const testInfo = this.testResults[index];
+      if (!testInfo) {
+        this.testStatistics.unranTests.push(index);
+      }
+      if (testInfo && !testInfo.passed) {
```

Comment:
`else if`?

Reply:
I decided to use early return. Let me know if you find this styl acceptable.